newIDE/app/src/Profile/Subscription/SubscriptionDialog.js

```diff
@@ -85,34 +86,39 @@ const styles = {
 };
 
 const cancelConfirmationTexts = {
-  title: t`Cancel your subscription`,
-  message: t`Are you sure you want to cancel your subscription?`,
-  confirmButtonLabel: t`Cancel my subscription`,
-  dismissButtonLabel: t`Go back`,
+  title: t`Cancel your subscription?`,
+  message: t`By canceling your subscription you'll be limited in cloud projects and cloud builds. Continue?`,
```

Comment:
"you'll be limited in cloud projects and cloud builds" 

that sounds a bit weak no? There are so many other things, I'm not sure if there are the main reason we want to highlight here?